2022q4/freebsd-foundation.adoc

```diff
@@ -34,9 +34,9 @@ In this Quarterly Status report you'll read about many of the areas we funded in
 Q4 to improve FreeBSD and advocate for the Project (the two main areas we spend
 money on). Check out reports on the internally and externally funded projects
 like Openstack on FreeBSD, Enabling Snapshots on Filesystems Using Journaled
-Soft Updates, FreeBSD as a Tier 1 cloud-init Platform, RISC-V support, and Arm64
```

Comment:
Why are you removing RISC-V support? According to the report riscv.adoc the work is indeed sponsored by The FreeBSD Foundation.